app/models/reception/resource_factory.rb

```diff
@@ -30,8 +28,10 @@ class ResourceFactory
     # @option request [Hash] sample: Hash containing the attributes for the sample
     # @option request [Hash] container: Hash containing the attributes for the container
     def request_attributes=(attributes)
-      @request_attributes = attributes.map do |request_attribute|
-        RequestFactory.new(resource_factory: self, reception:, **request_attribute)
+      @request_attributes = attributes.filter_map do |request_attribute|
+        rf = RequestFactory.new(resource_factory: self, reception:, **request_attribute)
+        # We only want to return the request if it's container does not already exist
+        rf.container.present? && rf.container.existing_records.present? ? false : rf
```

Comment:
I did think we could put this as a validator in RequestFactory but we don't want to filter by `valid?` here as that will hide requests failing on other things.